src/Diagnostics.RuntimeHost/Controllers/ProcessHealthController.cs

```diff
@@ -34,5 +36,12 @@ public async Task<IActionResult> HealthPing()
 
             return Ok("Server is up and running.");
         }
+
+        [HttpGet("/dependencyCheck")]
+        public async Task<IEnumerable<HealthCheckResult>> DependencyCheck()
```

Comment:
great addition :). This is something Me,Rekha and Ajay were discussing too last week for Liberation endpoints.